scripts/release/publish_release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -E
+set -o pipefail
+
+RELEASE_ID=$1
+
+GITHUB_URL=https://api.github.com/repos/${CODE_REPOSITORY}
+GITHUB_AUTH_HEADER="Authorization: Bearer ${GITHUB_TOKEN}"
+
+CURL_RESPONSE=$(curl -L \
+  -s \
+  --fail-with-body \
+  -X POST \
+  -H "Accept: application/vnd.github+json" \
+  -H "${GITHUB_AUTH_HEADER}" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "${GITHUB_URL}"/releases/"${RELEASE_ID}" \
+  -d '{"draft":false}')
```

Comment:
Should we also make it the latest release? (`"make_latest": "true"`)?

Reply:
It is not necessary as it becomes `latest` automatically: 
```specifies whether this release should be set as the latest release for the repository. Drafts and prereleases cannot be set as latest. Defaults to true for newly published releases. `
So I wouldn't set it unless you know of some specific conditions that require this option?